woocommerce-admin-test-helper.php

```diff
@@ -15,7 +15,10 @@
 /**
  * Register the JS.
  */
-function add_extension_register_script() {
+function add_wc_admin_helper_extension_register_script( $hook_suffix ) {
+	if ( 'tools_page_woocommerce-admin-test-helper' !== $hook_suffix ) {
+		return;
+	}
```

Comment:
Only `enqueue` the test-helper app js files when we are loading the test-helper page.
This way we don't load the JS files on any other wc-admin pages.